### PR TITLE
Add zero line

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Simple PR, one reviewer 0️⃣ 
Added horizontal line at 0 to aggregate graph so it's easier to see which changes are positive and which negative. Curious to see if others like it

### CODE REVIEW

**Comment by Oysters1874** [comment]
How about using the dashed line by adding line_dash="dash"?

**Comment by Oysters1874** [comment]
> line_dash="dash"

I am not sure which one looks better, curious about what others think.

**Comment by SashaWeinstein** [comment]
I tried the dashed line and didn't like it personally

**Comment by Oysters1874** [comment]
Cool, then we just use solid lines. It looks fine to me.

**Review by Oysters1874 — APPROVED** [review]

**Comment by Oysters1874** [comment]
Would you mind changing the base to dev. I didn't notice this pr is trying to merge into my previous one. I have already got that one merged in.

File: src/pluto/pluto.py [hunk 1]
```diff
@@ -478,14 +478,14 @@ def generate_graph(v1, v2):
...
-                diff=[v1[i] - v2[i]  for i in y]
+                diff = [v1[i] - v2[i] for i in y]
                 real_v1 = [v1[i] for i in y]
                 real_v2 = [v2[i] for i in y]
                 hovertemplate = "<b>%{x}</b> %{text}"
                 text = []
                 for n in range(len(y)):
                     text.append(
-                        "Diff: {} Current: {} Prev: {}".format(
+                        "Diff: {} | Current: {} | Prev: {}".format(
```
Comment (mbh329) [inline src/pluto/pluto.py:488]: Nice addition and compartmentalizes the text better

File: src/pluto/pluto.py [hunk 2]
```diff
@@ -504,6 +504,7 @@ def generate_graph(v1, v2):
             fig = go.Figure()
             fig.add_trace(generate_graph(v1, v2))
             fig.add_trace(generate_graph(v2, v3))
+            fig.add_hline(y=0, line_color="grey", opacity=0.5)
```
Comment (mbh329) [inline src/pluto/pluto.py:507]: Looks good!

**Review by mbh329 — APPROVED** [review]

File: src/pluto/pluto.py (hunk 1 above)
Comment (SashaWeinstein) [reply src/pluto/pluto.py:488]: oh shoot I didn't mean to add this lol. Do you like it? I decided against it but definitely open to discussion

**Pull request merged**